AIFG-3D-Connect-4/Grid.h

```diff
@@ -35,6 +35,15 @@ class Grid
 	/// </summary>
 	/// <param name="t_gameBoard">Game board</param>
 	void setGameBoard(GameBoard* t_gameBoard);
+
+	/// <summary>
+	/// Gets the cells for a grid.
+	/// </summary>
+	/// <returns>returns a vector of cells from a grid.</returns>
+	inline std::vector<sf::RectangleShape> getRects() { return m_rects; }
```

Comment:
Should return a const reference to avoid duplicating the vector each time its used